apps/web/components/EventDetails.tsx

```diff
@@ -0,0 +1,77 @@
+// EventDetails.tsx
+import React from "react";
+import GetTickets from "./GetTickets";
+import Image from "next/image";
+import { events } from "@/mock-data";
+
+const EventDetailsList: React.FC = () => {
+  return (
+    <div className="events-list">
+      {events.map((event) => {
```

Comment:
I think this component need to render details of one event. why is there a map here ?